src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,210 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ExpectedException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String EMPTY = "";
+    private static final String THREE_CHARACTERS = "abc";
+    private static final String FIVE_CHARACTERS = "abcdf";
+    private static final String SIX_CHARACTERS = "abcdef";
+    private static final String EIGHT_CHARACTERS = "abcdefas";
+    private static RegistrationService registrationService;
+    private static User user;
+    private static User newUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+        newUser = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user.setAge(20);
+        user.setLogin("userLogin");
+        user.setPassword("12345678");
+
+        newUser.setAge(32);
+        newUser.setLogin("userNewLogin");
+        newUser.setPassword("12345678New");
+    }
+
+    @Test
+    void register_userNull_notOk() {
```

Comment:
add some messages to notOk tests too 
like this "Test failed! Exception should be thrown if user is null!"